tracer/build/_build/Build.cs

```diff
@@ -162,6 +165,7 @@ void DeleteReparsePoints(string path)
     Target BuildNativeLoader => _ => _
         .Description("Builds the Native Loader, and publishes to the monitoring home directory")
         .After(Clean)
+        .OnlyWhenStatic(() => UseNativeLoader)
```

Comment:
Note for @dudikeleti and @GreenMatan @gleocadie and myself, who need to build the native loader as part of their day to day - we'd need to remember to set this parameter manually during local development. 

But as a temporary solution, I think that's fine.